interface/themes/core.css

```diff
@@ -7,67 +7,51 @@
  * @copyright Copyright (c) 2017 Robert Down
  * @license https://github.com/openemr/openemr/blob/master/LICENSE General Public License 3
  */
-
-@import "themeBuilder.php";
```

Comment:
This line has to stay, it allows users to override default theme fonts and font sizes